.github/workflows/go.yml

```
@@ -0,0 +1,23 @@
+name: Go
+
+on: [push]
+
+jobs:
+  Test:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-20.04
```

[Reference](https://github.com/kohofinancial/documentation/blob/main/best%20practices/github-actions.md#version-pinning-and-dependency-management)